pubsub/gcloud/aio/pubsub/subscriber.py

```diff
@@ -55,28 +56,33 @@ def cache_outdated(self) -> bool:
                 return True
             return False
 
+    async def _budgeted_queue_get(queue: 'asyncio.Queue[Any]',
```

Comment:
You can be a bit more specific with the type hints here: if you use a [TypeVar](https://docs.python.org/3/library/typing.html#typing.TypeVar) then you can say:
```
async def _budgeted_queue_get(queue: 'asyncio.Queue[T]', ..) -> List[T]
```
which in turn means you won't lose the response type value when calling this method.